filaments/3djake.json

```diff
@@ -0,0 +1,153 @@
+{
+    "manufacturer": "3DJAKE",
+    "filaments": [
+        {
+            "name": "{color_name}",
+            "material": "PETG",
+            "density": 1.24,
+            "weights": [
+                {
+                    "weight": 1000.0,
+                    "spool_weight": 231
+                }
+            ],
+            "diameters": [
+                1.75
+            ],
+            "extruder_temp": 240,
+            "bed_temp": 90,
+            "colors": [
+                {
+                    "name": "Weiß",
```

Comment:
Translate the color names to english please. Make sure you use the same names as the manufacturer used

Reply:
Done